### DISCUSSION TITLE
Subcomponent proptable tabs on MDX?

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#11481] [Help] [answered]
I've been looking for a way to render proptable tabs with MDX the way CSF does it:

```
export default {
  title: 'Components/Button',
  component: Button,
  subcomponents: { ButtonInfix, ButtonPrefix, ButtonSuffix },
};
```

![Screen Shot 2020-07-09 at 6 32 37](https://user-images.githubusercontent.com/40529625/87035001-4ee01500-c1ae-11ea-9f88-923b44c5e119.png)

I can't seem to find anything on the documentation. Is it possible?

**Comment by shilman** [comment, marked as answer]
Try `<Props components={{ Button, ButtonInfix, ButtonPrefix, ButtonSuffix }} />`

**Reply by ghost** [reply]
Sorry for the late reply. It works but I also needed to define them in the `Meta` component like so:

```
<Meta
  title="Components/Button"
  component={Button}
  subcomponents={{ ButtonInfix, ButtonPrefix, ButtonSuffix }}
/>
```
